buildspec_publish_to_sar.yml

```diff
@@ -6,17 +6,21 @@ phases:
       java: openjdk11
     commands:
       - pip3 install aws-sam-cli
+      - apt install -q -y jq
 
   build:
     commands:
       # Use AWS SAM to build and package the application by using AWS CloudFormation
-      - sam build  --debug
-      - sam package --debug --template-file .aws-sam/build/template.yaml --s3-bucket $S3_BUCKET --output-template-file sampackaged.yaml --force-upload
+      - sam build
+      - sam package --template-file .aws-sam/build/template.yaml --s3-bucket $S3_BUCKET --output-template-file sampackaged_raw.yaml
   post_build:
     commands:
+      # Getting latest version tag from git
+      - APPLICATION_VERSION=$(curl -s  https://api.github.com/repos/BIBSYSDEV/$GIT_REPO/releases/latest | jq -r '.tag_name')
```

Comment:
What happens when tags are not present?

Reply:
Pipeline fails... I belive that is what we want.... But the webhook triggers only on release event